pkg/debug/stop_debug.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package debug
+
+import (
+	ctx "context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func StopDebug(context *k8sutil.Context, clusterNamespace, deploymentName string) {
+
+	if !strings.Contains(deploymentName, "-debug") {
+		deploymentName = deploymentName + "-debug"
+	}
+
+	debugDeployment, err := verifyDeploymentExists(context, clusterNamespace, deploymentName)
+	if err != nil {
+		fmt.Printf("Missing mon or osd debug deployment name %s. %v\n", deploymentName, err)
+	}
+
+	fmt.Printf("removing debug mode from deployment %s\n", debugDeployment.Name)
+	err = context.Clientset.AppsV1().Deployments(clusterNamespace).Delete(ctx.TODO(), debugDeployment.Name, v1.DeleteOptions{})
+	if err != nil {
+		fmt.Printf("Error deleting deployment %s: %v", debugDeployment.Name, err)
```

Comment:
- A common error during deletion is that it doesn't exist. But let's ignore that error. 
- Need to exit after the failure?

Reply:
done